tools/xcodeproj_shims/installers/swiftmodules.sh

```diff
@@ -5,7 +5,7 @@ set -euo pipefail
 # Copy Bazel build `.swiftmodule` files to `DerivedData`. This is used by Xcode
 # and its indexing.
 echo "Start copying swiftmodules at `date`"
-FOUND_SWIFTMODULES=`grep -A 2 primary_output "$BAZEL_BUILD_EVENT_TEXT_FILENAME" | grep -w uri | grep "\.swiftmodule" | sed -e "s/uri: \"file:\/\///" -e "s/\"\$//" || true`
+FOUND_SWIFTMODULES=`grep -A 2 -E "important_output|primary_output" "$BAZEL_BUILD_EVENT_TEXT_FILENAME" | grep -w uri | grep -Eo "bazel-out.*\.swiftmodule" | sed -e "s/uri: \"file:\/\///" -e "s/\"\$//" | sort | uniq || true`
```

Comment:
This was missing outputs in many cases, added `important_output` to pick the missing ones and `sort | uniq` to avoid duplicates